packages/ripple-keypairs/test/codec.test.ts

```diff
@@ -43,11 +43,14 @@ describe('ripple-address-codec', function() {
     assert.equal(decoded2.type, 'secp256k1')
   })
 
-  it('can pass a type as second arg to encodeSeed', function() {
+  it('can pass a type as second arg to encodeSeed', () => {
     const edSeed = 'sEdTM1uX8pu2do5XvTnutH6HsouMaM2'
     const decoded = api.decodeSeed(edSeed)
     assert.equal(toHex(decoded.bytes), '4C3A1D213FBDFB14C7C28D609469B341')
     assert.equal(decoded.type, 'ed25519')
+    // @ts-expect-error -- We are testing ed25519 here
     assert.equal(api.encodeSeed(decoded.bytes, decoded.type), edSeed)
```

Comment:
Not sure if it's cleaner, but an alternative to the `@ts-expect-error` could be to cast (Ex.)
```suggestion
    assert.equal(
      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions -- decodedSeed's type is guaranteed to return this
      api.encodeSeed(decoded.bytes, decoded.type as 'ed25519' | 'secp256k1'),
      edSeed,
    )
```

Reply:
Thanks for the suggestion. I actually prefer the ts-expect-error

Reply:
The better solution would probably be to type `decodeSeed` so that the returned `type` is `'ed25519' | 'secp256k1'`

Reply:
Fixed